Wikipedia/Code/NSURL+WMFLinkParsing.m

```diff
@@ -23,6 +23,23 @@ + (NSURL *)wmf_URLWithDefaultSiteAndLanguageCode:(nullable NSString *)languageCo
     return [self wmf_URLWithDomain:WMFConfiguration.current.defaultSiteDomain languageCode:languageCode];
 }
 
++ (NSURL *)wmf_URLWithDefaultSiteAndContentLanguageCode:(nullable NSString *)contentLangaugeCode {
+    if (!contentLangaugeCode) return nil;
+    NSString *languageCode = nil;
+    NSString *languageVariantCode = nil;
+    if ([contentLangaugeCode containsString:@"-"]) {
```

Comment:
I don't think we should rely on the hyphen right away, there are [some language codes](https://github.com/wikimedia/wikipedia-ios/blob/main/Wikipedia/Code/wikipedia-languages.json#L1614) with a hyphen where this will break (i.e. I send in `zh-classical` language code, and expect to get a `zh-classical.wikipedia.org` URL.

Reply:
Thanks for catching that!

Reply:
I've changed the implementation so we don't need this method at all.